packages/core/test/actor.test.ts

```diff
@@ -1174,7 +1174,7 @@ describe('actors', () => {
       {
         actions: {
           setup: assign({
-            child: () => spawn(childMachine)
+            child: (_) => spawn(childMachine)
```

Comment:
Why doesn't strict mode like empty arguments?

Reply:
This is a long-standing issue with XState types - I don't understand it in full yet but TS is somehow inferring differently here when the parameter is used or not, even if it stays unused. This doesn't have that much in common with strict as it has to do with their inference algorithm. However, the strict mode changed assignability types related to function types and thus in here the inferred return stopped being assignable - the type that was previously computed here was already wrong here though.